prisma/schema.prisma

```diff
@@ -11,15 +11,31 @@ generator client {
 }
 
 model Company {
-  companyId        Int     @default(autoincrement()) @id
+  companyId        Int   @default(autoincrement()) @id
   name     String
+  users User[]
 }
 
 model User {
   userId        Int       @default(autoincrement()) @id
-  companyId     Int
+  companyId     Int? 
+  company   Company? @relation(fields: [companyId], references: [companyId])
   email         String?   @unique
   name          String
   password      String
   @@map(name: "users")
+}
+
+model Event {
+  eventId        Int       @default(autoincrement()) @id
+  date     DateTime
```

Comment:
can you add a `startDate` and an `endDate` instead?

Reply:
done